flows/final_state_determination.py

```diff
@@ -0,0 +1,81 @@
+# /// script
+# dependencies = ["prefect>=3.0.0"]
+# ///
+
+"""
+This example demonstrates flow final state determination based on task states.
+"""
+
+import time
+
+from prefect import flow, task
+
+# Imports for type annotations
+from prefect.futures import PrefectFuture, PrefectFutureList
+
+
+def wait_for_futures():
+    """
+    Wait for all assigned PrefectFuture and PrefectFutureList objects in the calling function's local scope. If you do not assign the return value of a task to a variable, it will not be waited for.
```

Comment:
Probably worth breaking this into two lines.

```suggestion
    Wait for all assigned PrefectFuture and PrefectFutureList objects in the calling function's local scope.
    If you do not assign the return value of a task to a variable, it will not be waited for.
```